fendermint/vm/interpreter/src/fvm/exec.rs

```diff
@@ -90,7 +90,11 @@ where
         let method_num = msg.method_num;
         let gas_limit = msg.gas_limit;
 
-        let apply_ret = state.execute_explicit(msg)?;
+        let apply_ret = if from == system::SYSTEM_ACTOR_ADDR {
```

Comment:
Ah it's okay, I do the same in the query interpreter because otherwise it threw an error, I just saw my comment there. Maybe a comment wouldn't hurt here either.